include/fixed_containers/fixed_robinhood_hashtable.hpp

```diff
@@ -190,10 +209,20 @@ class FixedRobinhoodHashtable
         // until we hit an empty one
         while (0 != bucket_at(table_loc).dist_and_fingerprint_)
         {
+            // update the backlink of the value pointed to by the bucket we're about to put in
+            // table_loc
+            bucket_for_value_index(bucket.value_index_) = table_loc;
```

Comment:
this is actually quite a lot of extra work that is required for every single insertion and deletion. We expect there to be a lot of buckets that need to be moved, and now each of those moves also requires touching this separate array.